requirements_csv.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Script to add versions to requirements.csv
+
+import csv
+import os
+import re
+from importlib.metadata import version
+
+REQ_CSV = os.path.abspath(os.path.join(os.path.dirname(__file__), "requirements.csv"))
+DOC_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "doc", "requirements.csv")
+)
+CACHE_CSV = os.path.join(
+    os.path.expanduser("~"), ".cache", "cve-bin-tool", "requirements.csv"
+)
+HTML_DEP_PATH = os.path.abspath(
+    os.path.join(
+        os.path.dirname(__file__), "cve_bin_tool", "output_engine", "html_reports", "js"
+    )
+)
+HTML_DEP_CSV = os.path.join(HTML_DEP_PATH, "dependencies.csv")
+
+
+def get_cache_csv_data(file):
+    data = []
+    with open(file, "r") as f:
+        r = csv.reader(f)
+        next(r)
+        for row in r:
```

Comment:
And maybe unpack row into `(vendor, product)` instead of using `row[0]`/`row[1]` in code.
I think this should work?
```suggestion
        for (vendor, product) in r:
```